python/cugraph/tests/test_leiden.py

```diff
@@ -0,0 +1,74 @@
+# Copyright (c) 2019, NVIDIA CORPORATION.
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+import gc
+import time
+
+import pytest
+
+import cugraph
+from cugraph.tests import utils
+
+# Temporarily suppress warnings till networkX fixes deprecation warnings
+# (Using or importing the ABCs from 'collections' instead of from
+# 'collections.abc' is deprecated, and in 3.8 it will stop working) for
+# python 3.7.  Also, these import community and import networkx need to be
+# relocated in the third-party group once this gets fixed.
+import warnings
+
+with warnings.catch_warnings():
+    warnings.filterwarnings("ignore", category=DeprecationWarning)
+
+
+def cugraph_leiden(cu_M, edgevals=False):
+
+    G = cugraph.Graph()
+    if edgevals:
+        G.from_cudf_edgelist(cu_M, source="0", destination="1", edge_attr="2")
+    else:
+        G.from_cudf_edgelist(cu_M, source="0", destination="1")
+    # cugraph Louvain Call
+    t1 = time.time()
+    parts, mod = cugraph.leiden(G)
+    t2 = time.time() - t1
+    print("Cugraph Time : " + str(t2))
```

Comment:
This is a pretty universal pattern through the existing python unit tests.  Perhaps rather than a FIXME we should just add a new issue requesting this be checked/updated for all existing unit tests?

Reply:
agreed, I was actually thinking of the other tests as well but figured we'd incrementally update them starting from somewhere, but creating an issue is better.  I created issue #1094 and I'll mark this as resolved.